docs/csharp/quick-starts/hello-world.yml

```diff
@@ -0,0 +1,171 @@
+  description: This tutorial teaches you C# interactively, using your browser to write C# and see the results of compiling and running your code. It contains a series of lessons that begin with a "Hello World" program. These lessons teach you the fundamentals of the C# language.
+  audience: Developer
+  level: Beginner
+  nextTutorialHref: working-with-numbers
+  nextTutorialTitle: Working with numbers in C#
+  displayType: two-column
+  interactive: csharp
+items:
+- durationInMinutes: 1
+  content: |
+    This tutorial teaches you C# interactively, using your browser to write C# and see the results of compiling and running your code. It contains a series of lessons that begin with a "Hello World" program. These lessons teach you the fundamentals of the C# language.
+- title: Run your first c# program
+  durationInMinutes: 2
+  content: |
+    Run the following code in the interactive window (type the following line in the window on the right and click the 'Run' button):
+
+    ```csharp
+    Console.WriteLine("Hello World!");
+    ```
+
+    Congratulations! You've run your first C# program. Now, replace the two lines above with these:
+- title: Declare and use variables
+  durationInMinutes: 3
+  content: |
+    Your first program printed the `string` "Hello World!" on
+    a screen. The rest of this lesson explores working with
+    the `string` type, which represents text in C#.
+
+    Your first program is limited to printing one message. You can write more
+    useful programs by using **variables**. A **variable** is a symbol you can
+    use run the same code with different values. Let's try it.
+
+    ```csharp
+    string firstName = "Bill";
+    Console.WriteLine(firstName);
+    ```
+
+    Those lines declare a variable, `name` and assigns it a value, "Bill". Then, it
+    prints out the name.
+
+    You can assign different values to any variable you declare. You can change
+    the name to one of your friends. Add these two lines in the interactive window above:
+
+    ```csharp
+    string firstName = "Bill";
+    Console.WriteLine(firstName);
+    firstName = "Maria";
+    Console.WriteLine(firstName);
+    ```
+
+    Notice that the same line of code prints two different messages, based on
+    the value stored in the `firstName` variable.
+
+    You may have also noticed the "Hello" was missing in the last two messages. Let's fix
+    that now.  Modify the last line to the following:
+
+    ```csharp
+    Console.WriteLine("Hello " + firstName);
+    ```
+
+    Click 'Run' again to see the results.
+- title: Work with strings
+  durationInMinutes: 4
+  content: |
+    Your last edit was our first look at what you can do with strings. Let's explore more.
+
+    The `+`, when applied to strings, creates a new string with the text from the first followed by the text from the second. This is referred to as *string concatenation*. You can combine any number of strings, including string variables.  Try this:
```

Comment:
I'd extend the introduction. Perhaps 
The `+` is called an operator in C#. When applied...